examples/calendar/backend/privacy/calendar_js/generate_witness.js

```diff
@@ -0,0 +1,20 @@
+const wc  = require("./witness_calculator.js");
+const { readFileSync, writeFile } = require("fs");
+
+if (process.argv.length != 5) {
+    console.log("Usage: node generate_witness.js <file.wasm> <input.json> <output.wtns>");
+} else {
+    const input = JSON.parse(readFileSync(process.argv[3], "utf8"));
+    
+    const buffer = readFileSync(process.argv[2]);
+    wc(buffer).then(async witnessCalculator => {
+	//    const w= await witnessCalculator.calculateWitness(input,0);
```

Comment:
please delete if not needed

Reply:
It's a pending task as stated in readme.

Reply:
since this is a WIP, it would be better to add a todo comment for the pending tasks here. The PR has so many changes, and it's not easy to review each file as per Readme.

Reply:
i agree with @Manik-Jain , please leave todos instead

Reply:
This is now completed. This file is generated by circom commands and not written manually.